roles/download/defaults/main.yml

```diff
@@ -690,24 +690,28 @@ runc_checksums:
     v1.1.2: 0
     v1.1.3: 0
     v1.1.4: 0
+    v1.1.7: 0
```

Comment:
Thanks @pomland-94

would you please add the 1.1.5, 1.1.6 together

Reply:
Yes i can do it.

Reply:
Done that